tests/Feature/Http/Controllers/LoginControllerResponseTest.php

```diff
@@ -157,16 +198,16 @@ public function testTokenSignedWithClientSecret(): void
         Config::set('oidc.client_id', 'test-client-id');
         Config::set('oidc.client_secret', 'the-secret-client-secret');
 
-        // Set current state, normally this is generated before logging in and send
-        // to the issuer, when the user is redirected for login.
+        // Set the current state, which is usually generated and saved in the session before login,
+        // and sent to the issuer during the login redirect.
         Session::put('openid_connect_state', 'some-state');
 
         // We simulate here that the user now comes back after successful login at issuer.
         $response = $this->getRoute('oidc.login', ['code' => 'some-code', 'state' => 'some-state']);
         $response->assertStatus(200);
         $response->assertJson([
             'userInfo' => [
-                'email' => 'teste@rdobeheer.nl',
+                'email' => 'tester@rdobeheer.nl',
```

Comment:
Maybe use the reserved example.com domain instead of an actual valid email address?
```suggestion
                'email' => 'tester@example.com',
```